src/js/components/Tag/stories/Size.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { Box, Tag, Text } from 'grommet';
+
+export const Size = () => (
+  <Box pad="large" direction="row" gap="medium" wrap>
+    {['xsmall', 'small', 'medium', 'large', 'xlarge'].map(size=>(
+      <Box gap="medium" align="start" key={size}>
+        <Text size={size} weight="bold">{size}</Text>
+        <Tag name="Name" value="Value" size={size}/>
+        <Tag value="Value" size={size}/>
+        <Tag value="Value" size={size} onRemove={()=>{}}/>
```

Comment:
`onRemove` for size "xsmall" seems bad for accessibility. The clickable target is only 12x12px. Wondering if we shouldn't support `onRemove` for this size OR at least advise away from it for Design System related patterns.

Reply:
Agree. I don't think xsmall makes sense to have in a case where you want onRemove. I bigger target may make xsmall not really any smaller than small if it does have onRemove. Either way these values were a guess just to get some discussion started :)

Reply:
Changed the remove button for xsmall to be like the small one. It seems to fit fine and looks decent.

Reply:
the `xsmall` remove looks better